content/en/docs/setup/additional-setup/cni/index.md

```diff
@@ -132,6 +132,11 @@ Some platforms required special installation settings.
           cniConfFileName: istio-cni.conf
           chained: false
     {{< /text >}}
+    
+{{< tip >}}
+Note: Using `sidecarInjectorWebhook` to inject the `k8s.v1.cni.cncf.io/networks: istio-cni` annotation will overwrite pre-existing annotations of the same name.
```

Comment:
Why are we documenting it if its not the right way to do it?

Reply:
We need to remove https://github.com/istio/istio/blob/e2c9358d20e56e03f7ce7c99e36690b17f5db607/manifests/profiles/openshift.yaml#L20 from the openshift profile which has existed for nearly 3 years since https://github.com/istio/istio/pull/28076 was merged

In the current docs for CNI it says you should set this up (all the way back to Istio 1.3). We do need to remove this reference as well in the OpenShift 4.2+ section @rrobb-aspen .

Reply:
I've updated this tip so that we highlight that using the `sidecarInjectorWebhook` can cause overwrites of the appended cni values for Openshift users who have it set, in case they run into issues but have removed the `sidecarInjectorWebhook` configurations to make it consistent with https://github.com/istio/istio/pull/45207